python/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+from unityagents import *
```

Comment:
We also need __init__.py files in the unitytrainers folder

Reply:
Added.

Reply:
I think you also need one in `unitytrainers/bc` and `unitytrainers/ppo`

Reply:
According to this: http://docs.python-guide.org/en/latest/writing/structure/#packages, init files are for packages.

Reply:
`ppo` is a package and we do use that package's modules outside in the `trainer_controller` module and also in the `test` package. 

Minor nitpick, we could rename the modules from (`ppo_models`, `ppo_trainer`) to (`models`,`trainer`) because it is under the `ppo` package